### DISCUSSION TITLE
Is there a widget for building forms?

### DISCUSSION THREAD

**Discussion by sminot** [Textualize/textual#289] [Q&A] [answered]
I'm trying to build a dynamic form which prompts for user input in a series of fields. Is there a particularly convenient way to do that with textual?

My apologies for posting such a general request for help. This library looks amazing and I'm trying to figure out how I can use it for my current project. It's kind of hard for me to find pointers on any discussion forums, so I thought I'd start here. Thanks!

[After posting, I found this https://github.com/Textualize/textual/discussions/105#discussion-3572797; there may not be any further update, but I thought I'd leave this post up in case there was]

I suppose another way to ask is, how would I go about rendering a dynamic display that has a series of description and text entry fields? In such a way that user modification of any of the entries would update a reactive element?

**Comment by aitoehigie** [comment, marked as answer]
This is the closest thing that handles your described usecase: https://github.com/sirfuzzalot/textual-inputs

**Reply by sminot** [reply]
Thank you for this!
